docs/releasenotes.md

```diff
@@ -5,6 +5,24 @@ title: Releasenotes
 
 # Release notes BAG Huidige bevragingen API
 
+## Versie 1.2.0 (22-06-2021)
+In deze release zijn enkele bugs in productie opgelost. Tevens zijn er enkele wijzigingen aan de API-specificaties en de feature-files aangebracht.
```

Comment:
Is het een idee om wat meer detail te geven wat er is aangepast? In issue 375 staat zoveel dat ik moeilijk kan vinden hoe er in de nieuwe versie wordt gesorteerd.

Reply:
Ik heb drie punten genoemd die ik op basis van de issues kon achterhalen. @strijm is er meer detail te leveren over de wijzigingen voor versie 1.2 ?

Reply:
Volgens mij klopt het wel wat er staat. Voor meer details staat er een mooi lijstje met opgeloste issues. De relasese is alleen nog niet compleet denk ik.